bpf/configs.h

```diff
@@ -6,5 +6,6 @@
 volatile const u32 sampling = 0;
 volatile const u8 trace_messages = 0;
 volatile const u8 enable_rtt = 0;
+volatile const u64 min_rtt = 50000; //50 micro seconds
```

Comment:
Thanks @dushyantbehl ! this will avoid having inconsistent values. We can iterate later to find a better solution. I don't despair of finding a good solution closer to your first implementation, which I found simpler but would need as you put it in this PR description :-)
 
If that's good for @msherif1234 , that's good for me too

Reply:
Thanks @jotak